dashboard/app/views/courses/index.html.haml

```diff
@@ -1 +1,6 @@
-%h1 Course Page
+- course = local_assigns[:course]
+
+- content_for(:head) do
+  %script{ src: minifiable_asset_path('js/courses/index.js'), data: {course_summary: course.summarize.to_json}}
```

Comment:
:mortar_board:  TIL in Haml `data-*` attributes [can be given as a hash](http://haml.info/docs/yardoc/file.REFERENCE.html#html5_custom_data_attributes).